portal/src/main/webapp/js/lib/vc-session.js

```diff
@@ -146,6 +146,17 @@ window.vcSession = window.vcSession ? window.vcSession : {};
       _selectedCases.push(_selectedCase);
       return _selectedCases;
     };
+    
+    var _generateCohortDescription = function(_cases) {
```

Comment:
Where is this coming from? There is already a function called `generateCohortDescription_`